README.md

```diff
@@ -1 +1,364 @@
+```none
+1 2	1 * * root /usr/bin/certbot renew --deploy-hook /home/ubuntu/starter-kit-storage-and-interfaces/scripts/cert-deploy --quiet --no-self-upgrade
+```
+
+## Deploying storage and interfaces
+Now that the storage and interfaces repository is cloned, start by editing the `config/config.yml` file (a sample can be found under `storage-and-interfaces/config` in this repository) and add the certificates you generated to the `app` entry (which is the download service), like:
+```yaml
+app: # this is for download
+  host: "0.0.0.0"
+  servercert: /shared/cert/fullchain.pem
+  serverkey: /shared/cert/privkey.pem
+  port: "8443"
+```
+
+And to the `s3inbox` entry in the `docker-compose.yml` file (a sample can be found under `storage-and-interfaces` in this repository) like:
+```yaml
+s3inbox:
+  environment:
+    - SERVER_CERT=/shared/cert/fullchain.pem
+    - SERVER_KEY=/shared/cert/privkey.pem
+```
+
+### Deploying auth
+Auth is a service in the storage-and-interfaces Docker compose file that enables the users to extract a token and an s3config file in order to download and upload data, respectively. 
+In the `docker-compose.yml` file, apart from the credentials of the LS AAI account (registered in earlier steps), you should also set the redirect URL, to something like:
+```yaml
+- ELIXIR_REDIRECTURL=https://login.gdi.nbis.se/elixir/login
+```
+
+### Storage
+The Swedish deployment is using an external S3 backend for storing the files. However, the Docker compose file in storage-and-interfaces contains a Minio instance, that can be used for archiving the data. In either case, set the credentials of the S3 backend in the `config/config.yaml` file, specifically the `S3AccessKey` and `S3SecretKey` values.
+
+### Update the issuer configuration
+In `config/iss.json` there exists a file that defines the issuer that will be used from the download service. Change this to point to your REMS instance, for example:
+```json
+{
+    "iss": "https://rems.gdi.nbis.se/",
+    "jku": "https://rems.gdi.nbis.se/api/jwk"
+}
+```
+
+## Deploying htsget
+To get the `htsget` working with TLS, move to the `start-kit-htsget` directory and edit the `htsget-config/config.json` file (a sample can be found under `htsget/config-htsget/` in this repository) and change the location of `serverCert` and `serverKey` to:
+```json
+"serverCert": "/shared/cert/fullchain.pem",
+"serverKey": "/shared/cert/privkey.pem"
+```
+
+And change the first source to point to the download endpoint. For example:
+```json
+{
+    "pattern": "^s3/(?P<accession>.*)$",
+    "path": "https://download.gdi.nbis.se/s3/{accession}"
+},
+```
+
+Then change the port where the service is mapped to the port you want to expose (currently set to 8081) by changing the `docker-compose-htsget.yml` file (a sample can be found under `htsget` in this repository) as follows:
+```yaml
+services:
+  server:
+    ports:
+      - 8081:3000
+```
+
+## Deploying REMS
```

Comment:
Maybe mention here that the REMS should be also registered in the LS AAI? Because in the section of the LS AAI above it mentions only that Auth service needs to be registered with it. 
Something like: "REMS needs to be registered with the LS AAI as well, similar to the Auth service. Both of them can be under the same service since you are allowed to have multiple Redirect URIs (one for the Auth and one for the REMS).
This information can also exist in the LS AAI registration section instead of here.